templates/etc/rsyslog.conf.j2

```diff
@@ -0,0 +1,61 @@
+# {{ ansible_managed }}
+#  /etc/rsyslog.conf	Configuration file for rsyslog.
+#
+#			For more information see
+#			/usr/share/doc/rsyslog-doc/html/rsyslog_conf.html
+#
+#  Default logging rules can be found in /etc/rsyslog.d/50-default.conf
+
+
+#################
+#### MODULES ####
+#################
+
```

Comment:
A hash with the name of the module as key and the configuration options for that module as an array with name/value pairs in the values?